apps/files_versions/lib/Command/ExpireVersions.php

```diff
@@ -73,7 +73,7 @@ protected function configure() {
 	protected function execute(InputInterface $input, OutputInterface $output): int {
 		$maxAge = $this->expiration->getMaxAgeAsTimestamp();
 		if (!$maxAge) {
-			$output->writeln("No expiry configured.");
+			$output->writeln("Auto expiration is configured - expiration will be handled automatically.");
```

Comment:
```suggestion
			$output->writeln("Auto expiration is configured - expiration will be handled automatically according to the expiration patterns detailed at the following link https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/file_versioning.html.");
```
Mentioned link https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/file_versioning.html :)

Reply:
done